### PR TITLE
Updating Entity Types

### PR DESCRIPTION
**Pull request by medhaanand**

### CODE REVIEW

File: docs/bot-builder/basic/entities.md [hunk 1]
```diff
@@ -245,11 +269,23 @@ $time - ‘tomorrow’</td>
   </tr>
 </table>
 
-#### **Text Entity**
+#### **Attachment Entity**
```
Comment (shrey4500) [inline docs/bot-builder/basic/entities.md:271]: Are you not mentioning what type of attachments we can collect?

File: docs/bot-builder/basic/entities.md [hunk 2]
```diff
@@ -245,11 +269,23 @@ $time - ‘tomorrow’</td>
...
+You would need to create a Button HSL for letting the user upload the attachment.
+
+![Entity Types](assets/bot-builder-user-says/entitytype.png)
+
+For the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+
+#### **Words & Phrases Entity**
 
-**Text** entity is meant to extract words/phrases from user utterances.
+**Words & Phrases** entity is meant to extract words/phrases from user utterances.
```
Comment (shrey4500) [inline docs/bot-builder/basic/entities.md:285]: Text entities can extract specific "text values" e.g name of a person, name of a city, etc from user message. Extracting word / phrase sounds a bit confusing. Doesn't it?

File: docs/bot-builder/basic/entities.md (hunk 1 above)
Comment (medhaanand) [reply docs/bot-builder/basic/entities.md:271]: Its there in the image. Check the netlify link

File: docs/bot-builder/basic/entities.md (hunk 2 above)
Comment (medhaanand) [reply docs/bot-builder/basic/entities.md:285]: Alright, changing it.

**Pull request merged**